### PR TITLE
feat: add update window

### LINKED ISSUE #645: Show an upgrade modal, like Spicetify itself does
### 📝 Provide a description of the new feature

Have a modal pop up when you click the "download new version" button that gets added to the toolbar if there's a new Marketplace version. 

It could have the changelog/release notes, or even just a title, link, and note about how to update marketplace. 

### ➕ Additional Information

For reference:
image

### PR DESCRIPTION
**Pull request by SunsetTechuila**
resolves #645

### CODE REVIEW

**Comment by SunsetTechuila** [comment]
image

File: src/styles/styles.scss [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
-@import url('./components/_grid.scss');
-@import url('./components/_dropdown.scss');
-@import url('./components/_card.scss');
-@import url('./components/_settings.scss');
-@import url('./components/_reload-modal.scss');
-@import url('./components/_add-snippet-modal.scss');
-@import url('./components/_readme-pages.scss');
-@import url('./components/_fixes.scss');
-@import url('./components/_import-export.scss');
+@use './components/prismjs-themes/prism-tomorrow.scss';
```
Comment (theRealPadster) [inline src/styles/styles.scss:9]: Ahh, this works now? I guess we were just doing it wrong?

File: .eslintrc.js [hunk 2]
```diff
@@ -12,11 +12,9 @@ module.exports = {
   ],
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true,
```
Comment (theRealPadster) [inline .eslintrc.js:16]: We don't need this anymore?

File: .eslintrc.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
-// eslint-disable-next-line no-undef
 module.exports = {
-  "ignorePatterns": ["node_modules", "dist"],
+  "ignorePatterns": ["node_modules", "dist", ".eslintrc.js", "spicetify.d.ts"],
   "env": {
     "browser": true,
-    "es2021": true,
+    "commonjs": true,
```
Comment (theRealPadster) [inline .eslintrc.js:5]: Do we need commonjs? We don't use `require()`

Comment (SunsetTechuila) [reply .eslintrc.js:5]: It is not only for "require"

File: .eslintrc.js (hunk 2 above)
Comment (SunsetTechuila) [reply .eslintrc.js:16]: We don't

File: src/styles/styles.scss (hunk 1 above)
Comment (SunsetTechuila) [reply src/styles/styles.scss:9]: When you use "import", sass does not add the contents of the imported file to the current file

File: .eslintrc.js (hunk 3 above)
Comment (SunsetTechuila) [reply .eslintrc.js:5]: But I can remove it if you want

**Review by theRealPadster — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**